doc/news/changes/incompatibilities/20221211Munch

```diff
@@ -0,0 +1,9 @@
+Fixed: The function ParamerHandler::add_parameter() used to 
+call the internal action. Since the action converts during
+that step the default value to a string and afterwards back,
+this could lead to rounding-off errors so that the default
+values might change in the case of floating-point numbers. 
```

Comment:
```suggestion
call the internal action. Within that step, the action
converts the default value to a string and back afterwards.
This can lead to round-off errors so that the default
values might change in the case of floating-point numbers. 
```